docs/docsite/rst/dev_guide/testing.rst

```diff
@@ -7,14 +7,15 @@ Testing Ansible
 .. contents:: Topics
    :local:
 
-Introduction
-============
 
-This document describes:
+Why test your Ansible contributions?
+====================================
+
+If you’re a developer, one of the most valuable things you can do is look at the GitHub issues list and help fix bugs, since we almost always prioritize bug fixing over feature development.
```

Comment:
Did you copy this text from a google doc? You'll need to replace all apostrophes and quote marks to get rid of the unicode quotes before the PR will pass Shippable.

Reply:
Got it, sorry about that!  Made appropriate edits and re-pushed to this PR.